sklearn/kernel_approximation.py

```diff
@@ -318,7 +318,7 @@ def fit(self, X, y=None):
         self : object
             Returns the transformer.
         """
-        check_array(X, accept_sparse='csr')
+        X = check_array(X, accept_sparse='csr')
```

Comment:
wouldn't `flake8` or LGTM.com complain if `X` is not used?

Reply:
It's used below, which is why I added it here?

Reply:
:man_facepalming: missed that, sorry.